scss/_variables.scss

```diff
@@ -1387,7 +1387,7 @@ $card-footer-color:                 $gray-700 !default; // Boosted mod
 
 // scss-docs-start accordion-variables
 $accordion-padding-y:                     $spacer * .5 !default;
-$accordion-padding-x:                     $spacer * .5 !default;
+$accordion-padding-x:                     0 !default;
```

Comment:
there's no change on visual accordion?

Reply:
It avoids to have a negative margin that didn't fit well with dark-accordions. IMO, it doesn't change anything for accordions that are on the same bg-color and correct some visual issues that could happen in other cases. 

However, I think that accordions need to have a small refresh because their implementation sounds weird to me in Boosted.

Reply:
I agree for the need of small refactoring and improvement for accordions to match the guidelines. Agree with this null value, but need to remove this line also : https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/b67299a8f0b03babc1fc94ec322883ed7162efa8/scss/_accordion.scss#L11